maker-frontend/src/MakerApp.tsx

```diff
@@ -173,7 +173,7 @@ export default function App() {
                                     makeNewCfdSellOrder(payload);
                                 }}
                             >
-                                {order ? "Update Sell Order" : "Create Sell Order"}
+                                {order ? "Update Maker Offers" : "Create Maker Offers"}
```

Comment:
Nit: we are in the maker frontend, so no need to mention this :) 

```suggestion
                                {order ? "Update Offers" : "Create Offers"}
```

Reply:
github has a bug and won't let me "commit suggestion" :( - I'll push a fixup!